test/1.4/errors/T_ESTATICEXPORT.dml

```diff
@@ -0,0 +1,27 @@
+/*
+  © 2022 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.4;
+
+device test;
+
+/// ERROR ESTATICEXPORT
+inline method m1(inline x) {}
+
+/// ERROR ESTATICEXPORT
+method m2(uint64 x) throws {}
+
+/// ERROR ESTATICEXPORT
+method m3() -> (uint64, uint64) {
+    return (0, 0);
+}
+
+method init() {
```

Comment:
what about methods inside arrays? they should also be banned?

Reply:
Right! I'll add a test for that.